GameData/RP-1/Science/Experiments/CrewScience/StationExperiments.cfg

```diff
@@ -0,0 +1,1013 @@
+    biomeMask = 0 
+    description = To monitor X-ray flux emitted by the Sun <b>NOTE: This experiment takes 30 Days.</b> //Placeholder Text. Based on MOL Proposal
+    mass = 0.0045 //Sourced from MOL Proposal Documentation
+    techRequired = start //Placeholder
+    cost = 0 //Placeholder
+    tags = spaceStationPrototypes
+    minCrew = 2 //Placeholder
+    celestialBodies = Earth
+    situations = ORBITING
+    RESULTS
+    {
+        default = Through extended observation, we can better predict when solar flares will occur based on the x-rays emitted.//Placeholder
+    }
+    KERBALISM_EXPERIMENT
+  	{
+		// sample mass in tons. if undefined or 0, the experiment produce a file
+		SampleMass = 0 //Placeholder
+		IncludeExperiment = 
+  	}
+}
+@PART[*]:HAS[#capsuleTier[StationPrototypeAndDevelopment]]:FOR[RP-0-Kerbalism]
+{
+	MODULE
+	{
+		name = Experiment
+		experiment_id = RP0solarXRayDetection
+		ec_rate = 0.01 //Sourced from MOL Proposal Documentation
+		data_rate = 1 //Placeholder
+		@data_rate /= 2592000 //30d
+		requires = CrewMin:1,Sunlight
+		resources = 
+		crew_operate = True
+		hide_when_unavailable = True
+    }
+}
+
+// ============================================================================
+// EarlyStation Experiments
+// ============================================================================
+
+// ============================================================================
+// Long Duration Space Habitation 2
+// ============================================================================
+EXPERIMENT_DEFINITION
+{
+    id = RP0longDurationHabit2
+    title = Multi-Month Space Habitation
+    baseValue = 1500 //Placeholder
+    scienceCap = 1500 //Placeholder
+    dataScale = 50 //Placeholder
+    requireAtmosphere = False
+    situationMask = 16 //Space Low
+    biomeMask = 0
+    description = To further analyze the health of humans in space over an extended period of time. <b>NOTE: This experiment takes 2 years and consumes 2500L of Research Payload every 30 days.</b> //Placeholder Text
+    mass = 0	//Placeholder
+    techRequired = earlySpaceStations
+    cost = 0
+    tags = earlySpaceStations
+    minCrew = 3
+    celestialBodies = Earth
+    situations = ORBITING
+    RESULTS
+    {
+        default = Through more extended habitation in Earth Orbit, we have learned more about the various physical and psychological health effects humans experience in space//Placeholder
+    }
+    KERBALISM_EXPERIMENT
+  	{
+		// sample mass in tons. if undefined or 0, the experiment produce a file
+		SampleMass = 0
+		IncludeExperiment = RP0longDurationHabit1
+  	}
+}
+@PART[*]:HAS[#capsuleTier[EarlyStation]]:FOR[RP-0-Kerbalism]
+{
+	MODULE
+	{
+		name = Experiment
+		experiment_id = RP0longDurationHabit2
+		ec_rate = 0.5
+		data_rate = 50 //Placeholder
+		@data_rate /= 63072000 //2 Years
+		requires = CrewMin:4
+		resources = ResearchPayload@0.0009645 //Consumes 2.5T of payload every 30 days. Placeholder
```

Comment:
4 crew should be changed to 3 crew.
2.5t of payload every 30 days is way too much. Resupplying 10 tons of payload every 120 days is entirely unreasonable. This amount should probably not be any higher than the prototype space stations. Even at 4 tons per 120 days it's still rather high.

A Progress resupply craft can only bring up 140d of supplies for three kerbals (oxygen, food, water) on a salyut 7. There would be no space left for any sort of research payload.

140d of every type of life support supply and research payload at a consumption rate of 1t of research payload per 30d would take just over 10kl, and weigh 8.21t in a SM-IV service module. Still higher than you could stuff in a Progress, meaning you'd need to be launching a progress about every 70 days to keep the station supplied, which seems excessive.

If you instead build a custom probe to supply this, and give it about 500m/s of dV, it'll weigh 10.5 tons, which doesn't seems more reasonable, but I still believe that the consumption rates, volumes, and masses of all consumables should be looked at to make supply via progress to a salyut-style station reasonable achievable.